servers/movie_writer/movie_writer.cpp

```diff
@@ -155,6 +155,7 @@ void MovieWriter::_bind_methods() {
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/movie_writer/mix_rate", PROPERTY_HINT_RANGE, "8000,192000,1,suffix:Hz"), 48000);
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/movie_writer/speaker_mode", PROPERTY_HINT_ENUM, "Stereo,3.1,5.1,7.1"), 0);
 	GLOBAL_DEF(PropertyInfo(Variant::FLOAT, "editor/movie_writer/mjpeg_quality", PROPERTY_HINT_RANGE, "0.01,1.0,0.01"), 0.75);
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/movie_writer/audio_bit_depth", PROPERTY_HINT_RANGE, "16,32,suffix:bit"), 16);
```

Comment:
This should be an enum instead of an integer value that directly specifies bit count, as only 16-bit and 32-bit are valid choices. (8-bit is also valid, but it's not relevant for MovieWriter usage.)

Reply:
I could make it an enum, but it seems kind of weird to use an enum (effectively an integer) to represent an integer, losing the flexibility to possibly insert other values (24bit, 8bit, etc.) later.  The hints prohibit entering anything besides 16 or 32.  I guess I could make the enum values actually 16 or 32, though, now that I think about it.

Reply:
Ok, updated this to be an enum from the user perspective.  Is that sufficient?